src/driver/main.cpp

```diff
@@ -396,7 +396,7 @@ struct loader
         std::ofstream fs;
         if(not output.empty())
         {
-            fs.open(output);
+            fs.open(output, std::ios::binary);
```

Comment:
Will this cause problems when we output as json or python?

Reply:
I checked for json, it's working fine. 
![command](https://github.com/ROCm/AMDMIGraphX/assets/147505839/68fecb13-7830-4d62-ab75-9bd74dbc8811)
![passed](https://github.com/ROCm/AMDMIGraphX/assets/147505839/8ee22267-bf08-43e9-9c56-2624592ab7fa)
But, need to check for python.

Reply:
With this change python works fine also.